api/doc/openapi.yaml

```diff
@@ -557,72 +514,80 @@ components:
           type: string
         diem_ledger_version:
           $ref: '#/components/schemas/LedgerVersion'
+    Uint64:
+      title: uint64
+      type: string
+      format: uint64
+      description: Unsiged int64 type value
+      example: "32425224034"
```

Comment:
Moved Uint64 definition up to here, because html UI renders schemas in order, and trying to let the simple & core types defined on top.